schema.json

```diff
@@ -0,0 +1,660 @@
+                "identify": {
+                    "type": "boolean",
+                    "description": "Determines whether identify is enabled for this layer.",
+                    "default": true
+                },
+                "singleEntryCollapse": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Indicates that the dynamic layer with a single layer entry should be rendered without the root group."
+                },
+                "layerEntries": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/$defs/dynamicLayerEntryNode"
+                    },
+                    "minItems": 1
+                }
+            },
+            "required": ["id", "layerType", "layerEntries", "url"],
+            "additionalProperties": false
+        },
+
+        "layerNode": {
+            "oneOf": [
+                {
+                    "$ref": "#/$defs/basicLayerNode"
+                },
+                {
+                    "$ref": "#/$defs/featureLayerNode"
+                },
+                {
+                    "$ref": "#/$defs/dynamicLayerNode"
+                }
+            ]
+        },
+
+        "extentNode": {
+            "type": "object",
+            "properties": {
+                "xmin": {
+                    "type": "number"
+                },
+                "ymin": {
+                    "type": "number"
+                },
+                "xmax": {
+                    "type": "number"
+                },
+                "ymax": {
+                    "type": "number"
+                }
+            },
+            "required": ["xmin", "ymin", "xmax", "ymax"],
+            "additionalProperties": false
+        },
+
+        "extentSetNode": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string"
+                },
+                "spatialReference": {
+                    "$ref": "#/$defs/spatialReferenceNode",
+                    "description": "The projection for the given extents"
+                },
+                "default": {
+                    "$ref": "#/$defs/extentNode",
+                    "description": "The default (starting) extent."
+                },
+                "full": {
+                    "$ref": "#/$defs/extentNode",
+                    "description": "The full extent (should give good view of the whole map, not necessarily the maximum extent); default will be used if not supplied."
+                },
+                "maximum": {
+                    "$ref": "#/$defs/extentNode",
+                    "description": "The maximum extent; full or default extents will be used if not supplied."
+                }
+            },
+            "required": ["id", "default", "spatialReference"],
+            "additionalProperties": false
+        },
+
+        "spatialReferenceNode": {

```

Comment:
same with this since it's tied to "extentSetNode"

Reply:
removed